consumergroup/consumer_group.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 		Initial           int64         // The initial offset method to use if the consumer has no previously stored offset. Must be either sarama.OffsetOldest (default) or sarama.OffsetNewest.
 		ProcessingTimeout time.Duration // Time to wait for all the offsets for a partition to be processed after stopping to consume from it. Defaults to 1 minute.
 		CommitInterval    time.Duration // The interval between which the processed offsets are commited.
+		ResetOffsets      bool          // call ResetOffsets() for the consumer group
```

Comment:
`Resets the offsets for the consumergroup so that it won't resume from where it left off previously.`